### PR TITLE
chore: bump version of robust crate

### PR DESCRIPTION
**Pull request by RobWalt**
- test ran through successfully
- see https://github.com/Stoeoef/spade/issues/89#issuecomment-1752799662 for benchmark results (5-10% improvements)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ bench = false
 
 [dependencies]
 smallvec = "1"
-robust = "0.2"
+robust = "1.1"
```
Comment (michaelkirk) [inline Cargo.toml:17]: minor nit:
```suggestion
robust = "1.1.0"
```

see https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277

**Review by michaelkirk — APPROVED** [review]
Again, I'm just a drive by reviewer, not a maintainer here.

Many of the [changes to the `robust` crate since v0.2](https://github.com/georust/robust/compare/0.2.3...v1.1.0) are maintenance changes that don't affect behavior, but there were some nice performance improvements which might account for the positive benchmarks you're seeing: 
https://github.com/georust/robust/pull/28



Also [3d operations were added](https://github.com/georust/robust/pull/23), but I don't think that's relevant to this crate.

File: Cargo.toml (hunk 1 above)
Comment (RobWalt) [reply Cargo.toml]: Thanks for the review! 😊

**Comment by Stoeoef** [comment]
Thanks a lot! Merging in once the clippy lint is fixed in #91

**Comment by RobWalt** [comment]
Thanks a lot!

**Pull request merged**